Package.swift

```diff
@@ -0,0 +1,55 @@
+// swift-tools-version: 5.9
+
+import PackageDescription
+
+let package = Package(
+    name: "maplibre-navigation-ios",
+    defaultLocalization: "en",
+    platforms: [.iOS(.v14)],
```

Comment:
is iOS 14 required for maplibre-native v6?

Reply:
Currently, MapboxDirections requires iOS 14 minimum

Reply:
Is this a new requirement? I have seen that you added SPM support there as well, but I think the old Carthage build was compatible with iOS 8 or something like that.

Has something changed that much that requires iOS14?

Reply:
Good point. I will check why we changed the deployment target to 14 for the directions framework.

Reply:
The MapboxDirections fork supports back to iOS 12.0. So I now restored iOS 12 support for this PR as well.